wazo_confd/plugins/user_blocklist/plugin.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2024 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from wazo_confd.helpers.types import PluginDependencies
+from .resource import (
+    UserMeBlocklistNumberItem,
+    UserMeBlocklistNumberList,
+)
+from .service import build_service
+
+
+class UserBlocklistPluginDependencies(PluginDependencies):
+    pass
```

Comment:
What is the use of this empty subclass, instead of using `Plugindependencies` directly?

Reply:
Yeah in that case not useful.